src/domain/project/schema.ts

```diff
@@ -85,11 +85,17 @@ export const singerSchema = z.object({
 });
 
 export const trackSchema = z.object({
+  name: z.string(),
   singer: singerSchema.optional(),
   keyRangeAdjustment: z.number(), // 音域調整量
   volumeRangeAdjustment: z.number(), // 声量調整量
   notes: z.array(noteSchema),
   pitchEditData: z.array(z.number()), // 値の単位はHzで、データが無いところはVALUE_INDICATING_NO_DATAの値
+
+  solo: z.boolean(),
+  mute: z.boolean(),
+  pan: z.number(),
+  gain: z.number(),
```

Comment:
ここだけ他と順番が違うのがちょっと気になりました。まあ問題はないのですが、気づいてしまったので。。
```suggestion
  gain: z.number(),
  pan: z.number(),
```